### PR TITLE
Added different courses in Tamil

### PR DESCRIPTION
**Pull request by JuanitaCathy**
## What does this PR do?
Add resources ie, Courses in Tamil

### Why is this valuable (or not)?
They have good reviews about the simplicity in their style of explaining.

### How do we know it's really free?

All the listed ones are from Youtube and hence, its completely free.

### For book lists, is it a book? For course lists, is it a course? etc.

Course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

**Comment by JuanitaCathy** [comment]
> For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

I've added playlist links with the name of the creator except few were complete course in a single video.

File: courses/free-courses-ta.md [hunk 1]
```diff
@@ -86,10 +88,17 @@ CSE Tamila by Eezytutorials
 * [Introduction to Machine Learning(Tamil)](https://www.youtube.com/playlist?list=PLyqSpQzTE6M-9thAeyB2mRFYvvW8AWxXX) - IIT Madras NPTEL
 * [Machine Learning in Tamil](https://www.youtube.com/playlist?list=PLJtSFa-YIedYu2QfQaHJJBLT096RxtMHD) - Majaa Matrix
 * [Machine Learning in Tamil](https://youtube.com/playlist?list=PL5itdT07Pm8wxRaPWljPntnBmnOs4ExDM) - Nithya Duraisamy
+* [Machine Learning with Python and R](https://www.youtube.com/watch?v=ucvXw1XUAy0&list=PL-1QQC56x1gEgj8C4L2hw5orryqgdnuoP) - Data Science Alive
```
Comment (eshellman) [inline courses/free-courses-ta.md:91]: for example use https://www.youtube.com/playlist?list=PL-1QQC56x1gEgj8C4L2hw5orryqgdnuoP
do the same with the others

Comment (JuanitaCathy) [reply courses/free-courses-ta.md:91]: Okay 👍🏽

**Comment by JuanitaCathy** [comment]
Hello, I've committed changes, thanks!

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**